.github/workflows/tagging-and-release.yml

```diff
@@ -0,0 +1,78 @@
+name: Tagging And Release
```

Comment:
In my opinion, there is no need to manually create tags and do GitHub releases as Changesets' action is doing that out of the box.
https://github.com/changesets/action#inputs by default the `createGithubReleases` is set to true.